index.html

```diff
@@ -1102,21 +1086,68 @@
           modifiers for particular payment method identifiers. For example, it
           allows you to adjust the total amount based on payment method.
         </dd>
+      </dl>
+    </section>
+    <section data-dfn-for="PaymentDetailsInit" data-link-for=
+    "PaymentDetailsInit">
+      <h2>
+        <dfn>PaymentDetailsInit</dfn> dictionary
+      </h2>
+      <pre class="idl">
+        dictionary PaymentDetailsInit : PaymentDetailsBase {
+          required PaymentItem total;
```

Comment:
Gecko hits [name duplication error](http://searchfox.org/mozilla-central/source/dom/bindings/parser/WebIDL.py#1867) while parsing member `total`. Not sure whether to revise WebIDL or adapt gecko to it.